### DISCUSSION TITLE
Adding a beep sound at the print finish.

### DISCUSSION THREAD

**Discussion by Jonny11bonk** [Jyers/Marlin#1032] [Ideas]
Hi! I have my Ender 3 V2 2 rooms next to me and I never know when the print is finished and I'm not planning to add a Raspberry Pi on it. Would it be possible to somehow trigger the display buzzer to beep when the print finishes?

**Comment by adam3654** [comment]
You could add a beep to the end g code of your slicer see the link below:
https://marlinfw.org/docs/gcode/M300.html
